test/server/http_filter_base_test.cc

```diff
@@ -0,0 +1,89 @@
+#include "server/http_dynamic_delay_filter.h"
+#include "server/http_test_server_filter.h"
+#include "server/http_time_tracking_filter.h"
+
+#include "test/server/http_filter_integration_test_base.h"
+
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+using namespace testing;
```

Comment:
Do we need this, since most symbols from the namespace are qualified? E.g. ::testing::Combine.

If you do prefer to shorten them, can we list them explicitly instead of subscribing to the entire testing namespace which *could* cause a conflict and confuse someone in the future?

E.g. for symbols that repeat a lot we can:

using ::testing::Combine;
using ::testing::ValuesIn;

Reply:
Done; added an anonymous namespace as well based on that it can only benefit us.